### PR TITLE
DLPX-65387 Upgrade rebooting screen is no longer observed upgrading from 5.3.5 to L

### PR DESCRIPTION
**Pull request by pzakha**
Adds upgrade reboot screen to migration upgrades, using dx_execute code from 5.3.

## TESTING
- manually verified that the reboot screen appears in the UI.
- git-ab-pre-push: http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/appliance-build-orchestrator-pre-push/1889/

### CODE REVIEW

**Comment by pzakha** [comment]
cc: @sebroy @sdimitro

**Comment by grwilson** [comment]
Do we need similar logic in our linux upgrade scripts?

**Comment by pzakha** [comment]
@grwilson, Not according to Michael, looking in the bug report.

> Appears to be specific to migration, since the correct screen is seen when upgrading L to L.

File: live-build/misc/migration-scripts/dx_execute [hunk 1]
```diff
@@ -51,6 +51,23 @@ EOM
...
 }
 
+function upgrade_alert() {
+	local pid
+
+	#
+	# This could hang if the JVM is in a bad state, so we spawn it in the
+	# background and give it 5 seconds to complete before giving up.
+	#
+	/opt/delphix/server/bin/jmxtool boot upgrade server &>/dev/null &
```
Comment (grwilson) [inline live-build/misc/migration-scripts/dx_execute]: Can we just use `timeout` instead? It would simplify this logic.

Comment (pzakha) [reply live-build/misc/migration-scripts/dx_execute]: Yeah, much simpler indeed.

**Pull request merged**